syntax = "proto3";
service robotOp {
  rpc Opmouse (OpRequest) returns (Reply) {}
}

message keys {
  string key = 1;
  bool ctrKey = 2;
  bool shiftKey = 3;
  bool altKey = 4;
}

message OpRequest {
  string mouseType = 1;
  int32 x = 2;
  int32 y = 3;
  keys keys = 4;
  int32 deltaX = 5;
  int32 deltaY = 6;
}



message Reply {
  string message = 1;
}
